### DISCUSSION TITLE
Input Shaping

### DISCUSSION THREAD

**Discussion by synman** [synman/Ender-3-S1-Pro-Firmware#22] [Features]
Details to follow as part of analysis

**Comment by synman** [comment]
This is now active in the bugfix-2.1.x builds.  If you do not intend on using it, I suggest you disable it (there is no way to zero it out currently via configuration as a default).
```
M593 F0  ; disable Input Shaping
```
Be sure to run through the calibration tests if you plan on using it @ https://marlinfw.org/docs/gcode/M593.html

I'm in contact with the IS author to see about how best to disable IS via configuration without disabling the capability entirely.

**Comment by synman** [comment]
got the answer I needed:  https://github.com/MarlinFirmware/Marlin/pull/24951#issuecomment-1421019148

future builds will have the Input Shaping capability enabled, but with a zeroed out frequency, essentially disabling it.

**Comment by stet345434543454345** [comment]
Is there a way to create an input shap8ng menu?
